### PR TITLE
parsing code added

### PR DESCRIPTION
**Pull request by lorusso7**
I add a test folder with the "parsing" code for Madgrpah cards. In addition inside the folder there are some card to test the code.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/parsing_code/dyellell012j_5f_NLO_FXFX_proc_card.dat~ [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/dyellell012j_5f_NLO_FXFX_proc_card.dat~]: can you please remove this file?

File: bin/MadGraph5_aMCatNLO/parsing_code/dyellell012j_5f_NLO_FXFX_run_card.dat~ [hunk 2]
```diff
@@ -0,0 +1,146 @@
+#***********************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/dyellell012j_5f_NLO_FXFX_run_card.dat~]: remove also this file

File: bin/MadGraph5_aMCatNLO/parsing_code/tt012j_5f_ckm_NLO_FXFX_proc_card.dat~ [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/tt012j_5f_ckm_NLO_FXFX_proc_card.dat~]: remove also this file

File: bin/MadGraph5_aMCatNLO/parsing_code/tt012j_5f_ckm_NLO_FXFX_run_card.dat~ [hunk 4]
```diff
@@ -0,0 +1,155 @@
+#***********************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/tt012j_5f_ckm_NLO_FXFX_run_card.dat~]: remove also this file

File: bin/MadGraph5_aMCatNLO/parsing_code/parsing.py [hunk 5]
```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+import sys, re, os, os.path
+import subprocess
+import string
+import os.path
+import urllib, re
+
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/parsing.py]: can you add a commented line to show how to execute the parser?

File: bin/MadGraph5_aMCatNLO/parsing_code/parsing.py [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+  print 'END of Parsing'
+  print "-----------------------------------------------------------------------"
+
+  proc_card.close() 
+  run_card.close() 
+
+
+#-------------------Run the function-----------------
+
+#link_pdfset = "https://lhapdf.hepforge.org/pdfsets.html"
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/parsing_code/parsing.py]: can you please cleanup these lines, if not useful?

**Comment by perrozzi** [comment]
could you consider moving the example datacards into a subfolder called "examples" or similar?

**Comment by lorusso7** [comment]
I applied the comments and suggestion. Do I open a new pull request or is possible update the #1154 ?

**Comment by perrozzi** [comment]
thanks, open a new one

**Pull request merged**